src/provisioning/Provisioner.ts

```diff
@@ -1,18 +1,37 @@
-import { Request } from "express";
-import { IrcBridge } from "../bridge/IrcBridge";
-import { Defer } from "../promiseutil";
-import { ConfigValidator, MatrixRoom, MatrixUser, PowerLevelContent, UserMembership } from "matrix-appservice-bridge";
-import Bluebird from "bluebird";
-import { IrcRoom } from "../models/IrcRoom";
-import { IrcAction } from "../models/IrcAction";
-import { BridgeRequest, BridgeRequestData } from "../models/BridgeRequest";
-import { ProvisionRequest } from "./ProvisionRequest";
-import logging, { RequestLogger } from "../logging";
-import * as promiseutil from "../promiseutil";
 import * as express from "express";
-import { IrcServer } from "../irc/IrcServer";
-import { IrcUser } from "../models/IrcUser";
-import { GetNicksResponseOperators } from "../irc/BridgedClient";
+import {ValidateFunction} from "ajv";
```

Comment:
I think we tend to pad with spaces when doing imports, is there a reason we've switched here?